config/eslint.cjs

```diff
@@ -56,7 +56,8 @@ module.exports = {
       'error',
       { argsIgnorePattern: '^_', varsIgnorePattern: '^_' },
     ],
-    '@typescript-eslint/no-redeclare': ['error'],
+    '@typescript-eslint/no-redeclare': 0,
```

Comment:
I don't think we have to set this down, when reading on [no-redeclare](https://eslint.org/docs/latest/rules/no-redeclare) we are not re*declaring* anyhting but only *reassiging*, which is totally norma, right?

I did a quick test re-adding the original rule setup and run`npm run lint` from root, still seems to work.